pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -343,6 +467,35 @@ func GetBootstrapClusterRoleBindings() []authorizationapi.ClusterRoleBinding {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: NodeRoleBindingName,
+			},
+			RoleRef: kapi.ObjectReference{
+				Name: NodeRoleName,
+			},
+			Groups: util.NewStringSet(NodesGroup),
+		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: NodeProxyRoleBindingName,
+			},
+			RoleRef: kapi.ObjectReference{
+				Name: NodeProxyRoleName,
+			},
+			// Allow node identities to run node proxies
+			Groups: util.NewStringSet(NodesGroup),
```

Comment:
If we're splitting out the roles, why not have our node certs include additional groups if they're doing more than just node work?